### PR TITLE
Change k8s-merge-robot to k8s-ci-robot

### PR DESCRIPTION
**Pull request by liubin**
Change `k8s-merge-robot` to `k8s-ci-robot`

### CODE REVIEW

File: contributors/devel/gubernator.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Gubernator runs on Google App Engine and fetches logs stored on Google Cloud Sto
 
 ### Test Failures list
 
-Issues made by k8s-merge-robot will post a link to a page listing the failed tests.
+Issues made by k8s-ci-robot will post a link to a page listing the failed tests.
```
Comment (cblecker) [inline contributors/devel/gubernator.md]: s/Issues/Comments

**Comment by liubin** [comment]
Hi @cblecker , I have replaced `Issues` by `Comments` at line 33 in file `contributors/devel/gubernator.md`

**Comment by cblecker** [comment]
Thanks for the contribution!

**Pull request merged**